Cargo.toml

```diff
@@ -12,7 +12,9 @@ members = [
   "experimental/trusted_shuffler/common",
   "experimental/trusted_shuffler/server",
   "experimental/trusted_shuffler/trusted_shuffler",
+  "experimental/uefi/client",
   "experimental/uefi/loader",
+  "experimental/uefi/proto",
```

Comment:
Does UEFI use protobuf?

Reply:
Yeah, it was a bad name (it's the loader that uses proto). However, I was able to get rid of the crate altogether so painting that bikeshed is now moot :)